papas/pfobjects.py

```diff
@@ -18,12 +19,12 @@ class PFObject(object):
     '''
 
 
-    def __init__(self, pfobjecttype=Identifier.PFOBJECTTYPE.NONE, subtype='u'):
+    def __init__(self, pfobjecttype=Identifier.PFOBJECTTYPE.NONE, subtype='u', value = 0.0):
```

Comment:
need docstring. what is value? name not explicit enough compared to the other parameters
idvalue below is a better name, but still need docstr